src/site/content/en/blog/handwriting-recognition/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Handwriting Recognition (WT)
```

Comment:
What's "WT"?

Reply:
Working title. @jpmedley, don't bother with the PR quite yet. I'm working with @christianliebel (one of our Google Developer Experts) and will only loop you in once the article is ready for your review. The ETA for the article is end of this week (May 14), so your review would probably only need to begin the week after (May 16).